systemd/systemd.go

```diff
@@ -86,7 +86,9 @@ var JournalctlCmd = jctl
 type Systemd interface {
 	DaemonReload() error
 	Enable(service string) error
+	EnableNow(service string) error
 	Disable(service string) error
+	DisableNow(service string) error
 	Start(service string) error
```

Comment:
I wonder if we could remove `Start`/`Stop` in favor of `StartNow`/`StopNow` (or the rename)?